### PR TITLE
[WFLY-5885] JBOSS_HOME validity check in jconsole sh, ps1 and bat scripts

### PR DESCRIPTION
**Pull request by lvydra**
https://issues.redhat.com/browse/WFLY-5885

### CODE REVIEW

File: ee-feature-pack/common/src/main/resources/content/bin/jconsole.bat [hunk 1]
```diff
@@ -46,6 +46,12 @@ rem Setup The Classpath
 
 set "CLASSPATH=%JAVA_HOME%\lib\jconsole.jar"
 set "CLASSPATH=%CLASSPATH%;%JAVA_HOME%\lib\tools.jar"
+
+if not exist "%JBOSS_HOME%\bin\client\jboss-cli-client.jar" (
+    echo   WARNING: Jar not found: %JBOSS_HOME%\bin\client\jboss-cli-client.jar
+    echo   WARNING: If this jar is missing, jconsole will fail to connect to Wildfly.
```
Comment (bstansberry) [inline ee-feature-pack/common/src/main/resources/content/bin/jconsole.bat:52]: s/to WildFly/to the server/g

I don't want to have to have add these to the EAP rebranding commits in https://github.com/jbossas/wildfly-to-product.

File: ee-feature-pack/common/src/main/resources/content/bin/jconsole.ps1 [hunk 2]
```diff
@@ -11,6 +11,12 @@ if (!$JAVA_HOME){
   exit
 }
 
+if (!(Test-Path $JBOSS_HOME\bin\client\jboss-cli-client.jar)) {
+    Write-Warning "Jar not found: $JBOSS_HOME/bin/client/jboss-cli-client.jar\"
+    Write-Warning "If this jar is missing, jconsole will fail to connect to Wildfly."
```
Comment (bstansberry) [inline ee-feature-pack/common/src/main/resources/content/bin/jconsole.ps1:16]: same as above

File: ee-feature-pack/common/src/main/resources/content/bin/jconsole.sh [hunk 3]
```diff
@@ -72,6 +72,12 @@ if $cygwin; then
     JAVA_HOME=`cygpath --path --windows "$JAVA_HOME"`
 fi
 
+if [ ! -f "$JBOSS_HOME/bin/client/jboss-cli-client.jar" ]; then
+    echo "WARNING Jar not found: \"$JBOSS_HOME/bin/client/jboss-cli-client.jar\""
+    echo "WARNING If this jar is missing, jconsole will fail to connect to Wildfly."
```
Comment (bstansberry) [inline ee-feature-pack/common/src/main/resources/content/bin/jconsole.sh:77]: same as above

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: ee-feature-pack/common/src/main/resources/content/bin/jconsole.bat (hunk 1 above)
Comment (rhusar) [reply ee-feature-pack/common/src/main/resources/content/bin/jconsole.bat]: @lvydra Also, if anything, we would stylize it as "WildFly" (just saying for future reference).

**Review by bstansberry — APPROVED** [review]

**Pull request merged**